pegasus/helpers/page_helpers.rb

```diff
@@ -88,3 +88,35 @@ def youtube_embed(youtube_url)
 
   %Q{<iframe title="YouTube video player" width="250" height="141" src="http://www.youtube.com/embed/#{youtube_id}" frameborder="0" allowfullscreen></iframe>}.html_safe
 end
+
+# A hacky method to generate locale-aware links to our lesson plans. This is
+# hacky primarily because we are forced to hardcode here a list of supported
+# languages; a list which we have no guarantee will remain accurate in the long
+# term.
+def hacky_localized_lesson_plan_url(path)
+  # This is the list of languages currently supported by the CurriculumBuilder
+  # i18n sync. This list should be kept in sync with the LANGUAGES settings
+  # variable in the curriculumbuilder project. (curriculumBuilder/settings.py)
+  curriculumbuilder_languages = [
+    'es-mx', 'es-es', 'hi-in', 'it-it', 'pl-pl', 'sk-sk', 'th-th'
+  ]
+
+  # This is a list of additional languages we want to support. These are
+  # languages for which there does exist content in that language on
+  # curriculum.code.org, but which aren't regularly synced.
+  # Be particularly cautious about adding languages to this list; not only is
+  # the content for that language not updated regularly, but new content is not
+  # added automatically. This means if you try to link to a recently-added
+  # lesson plan, it may not be there for any of these languages.
+  additional_languages = [
+    'et-ee', 'fr-fr', 'ko-kr', 'zh-tw'
+  ]
```

Comment:
@castro-jorge can you confirm that this is the correct set of languages?

Reply:
delete et-ee, and add: de-de (German), pt-br (Portuguese Brazil), tr-tr (Turkish), in-id (Indonesian, double check the locale), ar-ar (Arabic), zh-cn (Chinese Simplified)

Reply:
Note that we don't have translated content here for `pt-br`

Reply:
Hhm I am seeing translations for all HoC teacher, CSF Express, and CSF A-F teacher lesson plans through the in-context view. Where are you seeing that we do not have translated content

Reply:
the in-context view shows what's on crowdin; the distinguishing feature of these additional languages is that we don't sync them from crowdin.

See https://curriculum.code.org/pt-br/hoc/plugged/8/ for the actual content